docs/dev/installer.md

```diff
@@ -0,0 +1,188 @@
+        [--ingress-secret <secret>]             Will add ssl support to the ingress
+```
+
+## Install command
+
+The `install` command is used to install the Tekton Dashboard in a cluster. It supports both Kubernetes and OpenShift.
+
+Note that OpenShift supports two install methods:
+- Tekton Pipelines and Triggers installed by openshift pipelines operator
+- Tekton Pipelines and Triggers installed manually using YAML manifests
+
+Examples below illustrate the main `install` options:
+- [Installing on Kubernetes](#installing-on-kubernetes)
+- [OpenShift with Tekton Pipelines and Triggers installed by OpenShift Pipelines Operator](#openshift-with-tekton-pipelines-and-triggers-installed-by-openshift-pipelines-operator)
+- [OpenShift with Tekton Pipelines and Triggers installed manually using YAML manifests](#openshift-with-tekton-pipelines-and-triggers-installed-manually-using-yaml-manifests)
+- [Installing in a custom namespace](#installing-in-a-custom-namespace)
+- [Install ingress](#install-ingress)
+
+### Installing on Kubernetes
+
+To install the Tekton Dashboard on Kubernetes, using the default options, run the command:
+
+```bash
+./scripts/installer install
+```
+
+This will install the Dashboard in the `tekton-pipelines` namespace, assuming Tekton Pipelines and Triggers are installed in the `tekton-pipelines` namespace.
+
+To install the Dashboard in read only mode, add the `--read-only` option when calling the script:
+
+```bash
+./scripts/installer install --read-only
+```
+
+### OpenShift with Tekton Pipelines and Triggers installed by OpenShift Pipelines Operator
+
+To install the Tekton Dashboard on OpenShift after installing Tekton Pipelines and Triggers using OpenShift Pipelines Operator, run the command:
+
+```bash
+./scripts/installer install --openshift
+```
+
+If you want a read only Dashboard, add the `--read-only` option to the command line:
+
+```bash
+./scripts/installer install --openshift --read-only
+```
+
+### OpenShift with Tekton Pipelines and Triggers installed manually using YAML manifests
+
+To install the Tekton Dashboard on OpenShift after installing Tekton Pipelines and Triggers manually using YAML manifests, you will need to override Tekton Pipelines and Triggers install namespaces.
+
+When installing from the manifests, Tekton Pipelines and Triggeers will be deployed in the `tekton-pipelines` namespace, whereas OpenShift Pipelines Operator uses the `openshift-pipelines` namespace.
+
+Therefore, you will need to add the `--pipelines-namespace tekton-pipelines` and `--triggers-namespace tekton-pipelines` options when calling the installer script:
+
+```bash
+./scripts/installer install --openshift --pipelines-namespace tekton-pipelines --triggers-namespace tekton-pipelines
+```
+
+If you want a read only Dashboard, add the `--read-only` option to the command line:
+
+```bash
+./scripts/installer install --openshift --pipelines-namespace tekton-pipelines --triggers-namespace tekton-pipelines --read-only
+```
```

Comment:
Do we need to repeat this for each install option? I think it's already clear enough from the other examples.

Reply:
fixed